Steepfile

```diff
@@ -110,6 +110,7 @@ target :datadog do
   ignore 'lib/datadog/core/configuration/ext.rb'
   ignore 'lib/datadog/core/configuration/settings.rb'
   ignore 'lib/datadog/core/contrib/rails/utils.rb'
+  ignore 'lib/datadog/core/crashtracking/component.rb'
```

Comment:
I'm curious, was keeping the types here too annoying? The functions we're using seem reasonably simple 👀